Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/PhysicsSetupViewportUiCluster.h

```diff
@@ -39,15 +39,35 @@ namespace EMotionFX
             NumModes
         };
 
+        //! Used to track the cluster that a specific button is a part of.
+        struct ButtonData
+        {
+            AzToolsFramework::ViewportUi::ClusterId m_clusterId;
+            AzToolsFramework::ViewportUi::ButtonId m_buttonId;
+        };
+
+        constexpr static const char* const ColliderTranslationTooltip = "Switch to collider translation mode";
+        constexpr static const char* const ColliderRotationTooltip = "Switch to collider rotation mode";
+        constexpr static const char* const ColliderDimensionsTooltip = "Switch to collider dimensions mode";
+        constexpr static const char* const JointLimitParentRotationTooltip = "Switch to joint limit parent frame rotation mode";
```

Comment:
Nit: What about moving these just to where the tooltips are passed as function parameters? We probably don't have a second use case for them and the member types are quite verbose.